src/panels/config/integrations/integration-panels/zwave_js/zwave_js-logs.ts

```diff
@@ -114,6 +128,15 @@ class ZWaveJSLogs extends SubscribeMixin(LitElement) {
     );
   }
 
+  private _downloadLogs() {
+    const aEl = document.createElement("a");
+    aEl.download = `zwave_js ${this.startDateTime.toISOString()} to ${new Date().toISOString()}.log`;
```

Comment:
2 ISO date strings is going to generate a really long filename. I'm wondering if the timestamps are actually needed or if we should just use something like `zwave_js.log` let the users rename them post download or during the download (depending on if their browser is configured to prompt for filename/download location)